src/components/Clock/Clock.tsx

```diff
@@ -32,16 +27,16 @@ const getDegreeByDate = (m: Moment, hande: ClockHand): number => {
 };
 
 export default function Clock(props: ClockProps) {
-  const [modalOpen, setModalOpen] = useState(false);
+  // const [modalOpen, setModalOpen] = useState(false);
 
-  const handleButtonClick = () => {
-    setModalOpen(false);
-  };
+  // const handleButtonClick = () => {
+  //   setModalOpen(false);
+  // };
 
   return (
     <>
       <div className={styles.clockwrapper}>
-        <button className={styles.buttonclock} onClick={() => setModalOpen(true)}>
+        {/* <button className={styles.buttonclock} onClick={() => setModalOpen(true)}>
```

Comment:
These kind of comments tend to stay for ages and confuse people. Once the code is not needed just remove please.

Reply:
I'll fix it ok